bitwarden_license/src/Scim/Groups/PostGroupCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Scim.Context;
+using Bit.Scim.Groups.Interfaces;
+using Bit.Scim.Models;
+
+namespace Bit.Scim.Groups;
+
+public class PostGroupCommand : IPostGroupCommand
+{
+    private readonly IGroupRepository _groupRepository;
+    private readonly IGroupService _groupService;
+    private readonly IScimContext _scimContext;
+
+    public PostGroupCommand(
+        IGroupRepository groupRepository,
+        IGroupService groupService,
+        IScimContext scimContext)
+    {
+        _groupRepository = groupRepository;
+        _groupService = groupService;
+        _scimContext = scimContext;
+    }
+
+    public async Task<Group> PostGroupAsync(Guid organizationId, ScimGroupRequestModel model)
+    {
+        if (string.IsNullOrWhiteSpace(model.DisplayName))
+        {
+            throw new BadRequestException();
+        }
+
+        var groups = await _groupRepository.GetManyByOrganizationIdAsync(organizationId);
+        if (!string.IsNullOrWhiteSpace(model.ExternalId) && groups.Any(g => g.ExternalId == model.ExternalId))
+        {
+            throw new ConflictException();
+        }
+
+        var group = model.ToGroup(organizationId);
+        await _groupService.SaveAsync(group, null);
+        await UpdateGroupMembersAsync(group, model, true);
+
+        return group;
+    }
+
+    private async Task UpdateGroupMembersAsync(Group group, ScimGroupRequestModel model, bool skipIfEmpty)
+    {
+        if (_scimContext.RequestScimProvider != Core.Enums.ScimProviderType.Okta)
+        {
+            return;
+        }
+
+        if (model.Members == null)
+        {
+            return;
+        }
+
+        var memberIds = new List<Guid>();
+        foreach (var id in model.Members.Select(i => i.Value))
+        {
+            if (Guid.TryParse(id, out var guidId))
+            {
+                memberIds.Add(guidId);
+            }
+        }
+
+        if (!memberIds.Any() && skipIfEmpty)
```

Comment:
You can remove the `skipIfEmpty` param here, but otherwise leave this block in (because `skipIfEmpty` is always `true` at the moment)